SpectralOps/teller
# Feature/add gopass provider

Adding [Gopass](https://github.com/gopasspw/gopass) provider.
:warning: - this MR depended on #50 PR, i have created a new branch from this MR. we need merge the upgrade PR and then rebase from master. I'll do it once you will approve and merge the PR

Test:

1. make sure you have Gopass installed
2. run `teller new` with Gopass provider
3. edit gopass section with right secret path


Read:
```sh
 go run main.go show
```

Update:
```sh
go run main.go put PATH-1=**** PATH-2=**** --providers gopass --sync
```
Test:
Generate Gopass client:
```sh
mockgen -source pkg/providers/gopass.go -destination pkg/providers/mock_providers/gopass_mock.go
```

There is also an open issue #11
